src/Soneta.Sdk/Sdk/Sdk.targets

```diff
@@ -55,7 +55,7 @@
           Command="&quot;@(SonetaGeneratorExe)&quot; -s &quot;@(SonetaSchemaReference,'&quot; &quot;')&quot; @(SameAssemblyBusinessXmls,' ') &quot;%(WorkingXmls.FullPath)&quot;" />
 
     <ItemGroup>
-      <GeneratedCs Include="@(WorkingXmls->'%(RelativeDir)%(Filename).cs')"/>
+      <GeneratedCs Include="@(WorkingXmls->'%(RelativeDir)%(Filename).cs')" Pack="false"/>
```

Comment:
Interesuje mnie dlaczego Pack się ustawił na true, czyżby odziedziczył to po iterowanej kolekcji?
Co by było, jakby na Compile w Remove i Include wpisać bezpośrednio @(WorkingXmls->'%(RelativeDir)%(Filename).cs')?

Reply:
Jak wpiszę bezpośrednio, to jest ten sam efekt co przedtem czyli Pack ustawia się na true